tesp/workflow.py

```diff
@@ -28,8 +31,10 @@
 from eugl.fmask import fmask
 
 
-ERROR_LOGGER = wrap_logger(logging.getLogger('ard-error'),
+ERROR_LOGGER = wrap_logger(logging.getLogger('wagl-error.packaging'),
```

Comment:
tesp-error might be more applicable than wagl-error, unless it helps line up with the wagl module logging.